wiki/Beatmap/en.md

```diff
@@ -1,38 +1,52 @@
 
-Visual components of a beatmap, contrary to its patterns, provide auxiliary experience during gameplay and can be disabled in the [visual settings](/wiki/Visual_Settings) overlay for the most part. Those include:
+## Beatmap submission
 
-- [background imagery](Background) and video, which both can act as a playfield;
-- [kiai time](/wiki/Kiai_time) and [combo colours](/wiki/Glossary/Combo_colour) that help distinguish different beatmap sections from each other;
-- [storyboard](/wiki/Storyboards) that adds motion graphics and special effects, or provides a background story or theme for the song;
-- [custom skin](/wiki/Skinning), which changes the appearance of interface and gameplay elements;
-- [title text](Title_Text), which presents the song's metadata.
+*Main article: [Submission](/wiki/Submission)*
 
-## Beatmapsets
+Beatmap authors can [submit](/wiki/Glossary#bss) their creations to the public [beatmap listing](https://osu.ppy.sh/beatmapsets). While every beatmap is attributed to [a single person](/wiki/Beatmap/Beatmapsets/Beatmapset_host), it is often a joint effort: some difficulties may be mapped by other people working [in collaboration](/wiki/Beatmap/Beatmap_collaborations) or [separately](/wiki/Beatmap/Beatmapsets/Guest_difficulty).
 
-*Main article: [Beatmapsets](Beatmapsets)*
+<!-- TODO: after https://github.com/ppy/osu-web/issues/5852 is resolved, this section will need an update -->
 
-On a higher level, beatmaps are grouped into [beatmapsets](Beatmapsets), and then [submitted](/wiki/Glossary#bss) to the [beatmap listing](https://osu.ppy.sh/beatmapsets). Each beatmapset typically covers one song and holds together several beatmaps with different object placement and gameplay settings. In this context, a beatmap is called a [difficulty](/wiki/Difficulties).
+After the submission, a beatmap gains additional metadata fields, such as description, language, genre, and the explicit content marker, which can be changed by the beatmap author themselves on the web site. The beatmap also acquires [title text](Title_Text), appearance of which may be altered with the aid of [NAT](/wiki/People/The_Team/Nomination_Assessment_Team).
 
-While every beatmapset is attributed to [a single person](Beatmapsets/Beatmapset_host), it is often a joint effort: some difficulties may be done by other people working [in collaboration](Beatmap_collaborations) or [separately](Beatmapsets/Guest_difficulty).
+### Beatmap identification
+
+Every submitted beatmap is assigned a numeric identifier, or `BeatmapSetID`, by which it can be tracked on the web site and via [osu!api](/wiki/osu!api). Difficulties of a beatmap also have their own numeric identifiers, called `BeatmapID`. Both of these numbers may be seen in a URL leading to a specific difficulty's tab on a beatmap's page. It has the following format:
```

Comment:
"it" is a little ambiguous
```suggestion
Every submitted beatmap is assigned a numeric identifier, or `BeatmapSetID`, by which it can be tracked on the web site and via [osu!api](/wiki/osu!api). Difficulties of a beatmap also have their own numeric identifiers, called `BeatmapID`. Both of these numbers may be seen in the URL leading to a specific difficulty's tab on a beatmap's page. Said URL has the following format:
```